Rakefile

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+ENV['SKIP_SOLIDUS_BOLT'] = 'true'
```

Comment:
I think there's no need for that, as it'll be already skipped if there's no Gemfile, as in the case of the dummy app: https://github.com/solidusio/solidus_frontend/blob/aaf0f7db868c69866036dbdd09484783e70907ab/lib/generators/solidus_frontend/install/install_generator.rb#L46